hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -18,17 +23,18 @@
     },
   ],
   wakeup_list: [
-    { name: "aon_timer",
+    { name: "aon_timer_wkup_req",
       desc: "Raised if the wakeup or watchdog timer has hit the specified threshold"
     },
   ],
   inter_signal_list: [
-    { name:    "counter_run",
-      type:    "uni",
-      act:     "rsp",
-      package: "",
-      struct:  "logic",
-      width:   "1"
+    // Broadcast from LC
+    { struct:  "lc_tx"
```

Comment:
@msfschaffner 
could we ever have a case where we somehow skip escalate and go directly into virtual scrap?
It's technically possible by software right?  Just wondering if we'll need more conditions here.

Reply:
yes this is technically possible, since SW may configure the alert handler to not trigger this escalation action, and go to scrap directly (although that would not be recommended).

if we do not want to rely on SW to do this correctly, we have a couple of options:
1. make sure lc_escalate_en is also triggered when scrapping the state
1. send out a separate lc signal that indicates we are in scrap state
1. send out a dedicated watchdog disable life cycle signal

WDYT?

Reply:
Up to you guys, but 1. sounds sensible to me!

Reply:
yeah 1 sounds good to me as well.

Reply:
yeah that would be great, thanks!